### PR TITLE
Update testing in controllers

### PR DESCRIPTION
**Pull request by CheezItMan**
## Description
Before, when we did controller testing students knew about fixtures and validations.  Now we need to teach them controller testing without either of those.  So I've updated the lecture (and fixed the missing objectives) to match our new week 1 plans.  

## Feedback Requested
Please utilize the built-in GitHub Reviewers functionality to assign necessary reviewers. Use this section to further specify portions of the PR that you would like additional or focused feedback on.

### CODE REVIEW

File: 08-rails/testing-db-in-controllers.md [hunk 1]
```diff
@@ -49,21 +63,31 @@ describe "create" do
...
+    expect(book.author).must_equal book_hash[:book][:author]
+    expect(book.description).must_equal book_hash[:book][:description]
   end
 end
 ```
 
 The example above illustrates that the test can pass in a mock-params hash into the request with the `params: params_hash` argument.  Notice we verify that the number of books in the database increases and that the last book in the database has the correct title, author and description.
 
-**Exercise** You also need to create a test in which the params are invalid, violating validations.  With a partner write another test in which the params are invalid.  You can see an example [here.](code_samples/create_controller_test.rb)
+**Exercise** You also need to create a test in which the params are invalid or missing.  With a partner write another test in which the params are invalid.  We have an [example solution](code_samples/create_controller_test.rb)
```
Comment (droberts-ada) [inline 08-rails/testing-db-in-controllers.md]: If the params are missing, will that cause a problem? Remember that we have no validations for this model.

Comment (CheezItMan) [reply 08-rails/testing-db-in-controllers.md]: There wont' be a `:book` field in params and strong params should fail causing an exception.  We could still take this test out.

Comment (droberts-ada) [reply 08-rails/testing-db-in-controllers.md]: Ah, I see. I like the idea then, it'll be good to have a test for a failure case.

Will we have introduced strong params at this point in the week? Looks like we don't cover that until Friday?

Comment (CheezItMan) [reply 08-rails/testing-db-in-controllers.md]: I'll pull it out for Wednesday and add it back in on Friday.

**Pull request merged**